translations/fr/try_ruby_170.md

```diff
@@ -0,0 +1,37 @@
+---
+lang:   EN
+title:  Élementaire
+answer: ^(12|35|47)$
+load:   ticket = [12, 47, 35]
+ok:     D'accord
+error:  Presque
+---
+
+Donc nous avons stocké nos numéros de lotterie dans la variable ticket.
+Comment faire pour les retrouver?
+
+On a déjà vu qu'il est possible d'obtenir le numéro le plus élevé avec __max__.
+On peut également obtenir le premier (__first__) ou le dernier (__last__)
+élement de la liste.
+Mais que faire si on a besoin d'un élement specifique de la liste?
+
+### [ ]
+Ruby utilises des crochets droits [ ] pour viser un élement.
+Les crochets droits sont très commun dans Ruby.
+Ce sont comme des lunettes de visée pour aligner une cible. Exactement.
+Ces crochets signifient, "Je cherche ____". Prêt, vises.
```

Comment:
I feel that `Prêt, vises` is much less idiomatic than the english `Ready, aim`. I would have removed it, or used something simpler such as. `Pan !` or `En plein dans le mille !` or both.

Reply:
True, that was awkward.